### DISCUSSION TITLE
Redesign FREE-HIT

### DISCUSSION THREAD

**Discussion by priyankarpal** [JasonDsouza212/free-hit#88] [Ideas]
FREE-HIT is a fantastic website. People save their time by using it & also bookmarking it.

So we are thinking to redesign the whole FREE-HIT website.

**Comment by suzy-g38** [comment]
That's true. I have already saved some websites from there. Such an amazing project must say. 
If you guys are thinking about redesigning, I want to be a part of that.

**Reply by priyankarpal** [reply]
> That's true. I have already saved some websites from there. Such an amazing project must say. If you guys are thinking about redesigning, I want to be a part of that.

cool

**Comment by JasonDsouza212** [comment]
Yes, redesigning it makes sense, but it should also have many other components such as mobile nav , List and grid view and so on

**Reply by priyankarpal** [reply]
> Yes, redesigning it makes sense, but it should also have many other components such as mobile nav , List and grid view and so on

don't worry we will add these. Just provide us the colour details.

we will also migrate to Tailwind css

**Comment by JasonDsouza212** [comment]
Okay, I'll finalize the colors. 
And who will be working on this?

 Before implementing, let's finalize the UI after I finalize the colors, and then we can proceed further

**Reply by priyankarpal** [reply]
ok

**Reply by suzy-g38** [reply]
> Okay, I'll finalize the colors. And who will be working on this?
> 
> Before implementing, let's finalize the UI after I finalize the colors, and then we can proceed further

I also want to add that let's use a database. Because I don't think storing data in a JSON format can help us in a long run.

**Reply by JasonDsouza212** [reply]
> > Okay, I'll finalize the colors. And who will be working on this?
> > Before implementing, let's finalize the UI after I finalize the colors, and then we can proceed further
> 
> I also want to add that let's use a database. Because I don't think storing data in a JSON format can help us in a long run.

Let's consider using a database when there are many daily users. For now, we can store the data in a JSON file

**Comment by priyankarpal** [comment]
Also, we can add some separate pages for websites such as `HomePage`, `LinksPage`, `DocumentationPage` etc. What is your opinion? @JasonDsouza212 @suzy-g38

**Comment by JasonDsouza212** [comment]
and also a page to bookmark the tools

**Comment by JasonDsouza212** [comment]
for now we can use local storage to do that

**Reply by priyankarpal** [reply]
So, we need to upgrade the sites step by step.

**Reply by priyankarpal** [reply]
> for now we can use local storage to do that

but if the user clears their browser data or switches devices, they may lose their bookmark data.

**Comment by JasonDsouza212** [comment]
Yup designing it is the first thing we need to do, or creating pages

**Reply by priyankarpal** [reply]
Who will design? 🥲 I don't have enough knowledge. 

Also share the demo 🥲

**Reply by suzy-g38** [reply]
> Who will design? 🥲 I don't have enough knowledge.
> 
> Also share the demo 🥲

Same here I don't have knowledge of design.

**Reply by priyankarpal** [reply]
> > Who will design? smiling_face_with_tear I don't have enough knowledge.
> > Also share the demo smiling_face_with_tear
> 
> Same here I don't have knowledge of design.

😢

**Reply by JasonDsouza212** [reply]
> Who will design? 🥲 I don't have enough knowledge.
> 
> Also share the demo 🥲

I know a little about designing, But one of my friends is good at designing Ill ask him to do it

**Comment by suzy-g38** [comment]
One thing I want to add though, to discuss this now it's pretty early but still I want to share, Please consider adding a feature where a user or contributor can like or dislike a tool. I think that will give this project some extra dimension. 
I am again saying, this may be not for now as we can't use any database but consider it as a future addition when there will be a dedicated database.

**Reply by priyankarpal** [reply]
> One thing I want to add though, to discuss this now it's pretty early but still I want to share, Please consider adding a feature where a user or contributor can like or dislike a tool. I think that will give this project some extra dimension. I am again saying, this may be not for now as we can't use any database but consider it as a future addition when there will be a dedicated database.

yes. 💯

**Reply by JasonDsouza212** [reply]
> One thing I want to add though, to discuss this now it's pretty early but still I want to share, Please consider adding a feature where a user or contributor can like or dislike a tool. I think that will give this project some extra dimension. I am again saying, this may be not for now as we can't use any database but consider it as a future addition when there will be a dedicated database.
Yes its a good feature to add, that can help us in sorting it by popularity

**Reply by suzy-g38** [reply]
> > One thing I want to add though, to discuss this now it's pretty early but still I want to share, Please consider adding a feature where a user or contributor can like or dislike a tool. I think that will give this project some extra dimension. I am again saying, this may be not for now as we can't use any database but consider it as a future addition when there will be a dedicated database.
> > Yes its a good feature to add, that can help us in sorting it by popularity

Yup another good addition

**Comment by aashay28** [comment]
Hello, could you please assign me the task of redesigning the website? Thank you.

**Comment by JasonDsouza212** [comment]
I have some design ideas so I want some help I creating an about page

**Reply by aashay28** [reply]
Kindly share the ideas and lets build it !

**Comment by aashay28** [comment]
Redesign has been done after four hours of effort.
- tailwindcss

![Screenshot 2023-04-26 at 5 59 51 PM](https://user-images.githubusercontent.com/108337259/234576938-5871f6ab-2b44-47c7-9e3b-573814aa6e2a.png)

**Comment by JasonDsouza212** [comment]
That looks good, but I was thinking of something more like a Notion-style theme.

**Comment by JasonDsouza212** [comment]
https://www.figma.com/file/gGFYksGQWFIkvcCzBTpPCD/FH-(1)?node-id=0%3A1&t=CW34ZNANeUDAIWnz-1

Follow this link and you can make suggestions

**Reply by aashay28** [reply]
### Suggestions

- Left side bar must be open if it's not a small screen.
- Search bar should be display if the screen is small it should display icon.
- Font size is too big of tags.
- Need an alternative way to display a card.

**Comment by JasonDsouza212** [comment]
Is anyone interested in designing the about page based on the UI I have provided?

**Reply by priyankarpal** [reply]
ok give me the details

**Reply by suzy-g38** [reply]
> ok give me the details

Hey @priyankarpal can I work with you on this part?

**Reply by priyankarpal** [reply]
yeah sure

**Comment by JasonDsouza212** [comment]
I believe I have emailed you the Figma design. Can you confirm?

**Reply by priyankarpal** [reply]
yeah i have

**Comment by JasonDsouza212** [comment]
Should I close this discussion since we have a new design? If you have anything else to add, I can keep this discussion open.

**Reply by priyankarpal** [reply]
yeah
